l10n/en/firefox/browsers/compare/index.ftl

```diff
@@ -27,15 +26,27 @@ compare-index-how-well-does-your-browser = How well does your browser work acros
 compare-index-almost-all-of-the-browsers = Almost all of the browsers compared here allow synchronization between desktop and mobile devices. You’ll need an account to do it, which you can use to log into the browser on all devices and synchronize things like passwords, browsing history, bookmarks and settings.
 compare-index-conclusion = Conclusion:
 compare-index-and-the-winner-is = And the winner is…
+
+# Obsolete string
 compare-index-find-out-how-firefox-chrome = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera }, { -brand-name-brave } and { -brand-name-ie } differ in terms of privacy, utility and portability.
-compare-index-find-out-how-firefox-fallback = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-ie } differ in terms of privacy, utility and portability.
+
+compare-index-find-out-how-firefox-chrome-v2 = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-brave } differ in terms of privacy, utility and portability.
+
+# Obsolete string
 compare-index-looking-for-a-better-browser = Looking for a better browser? We’ll compare { -brand-name-firefox } with { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera }, { -brand-name-brave } and { -brand-name-ie } to help you make your decision.
-compare-index-looking-for-a-better-fallback = Looking for a better browser? We’ll compare { -brand-name-firefox } with { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-ie } to help you make your decision.
```

Comment:
I removed some older `-fallback` FTL string IDs that are now being replaced w/ new fallbacks. These were 4+ years old so should be fine to remove